### ISSUE TITLE
update 3d party library and tag

### ISSUE THREAD

**Issue by taraghi** [projectestac/moodle-local_oauth#17]
Hi,

would you mind 
- updating the applied 3d party plugin (https://github.com/bshaffer/oauth2-server-php/) to the latest release (currently v1.12.1)
- and tagging your releases (Our build process bases on tags. Consequently we do need a tag to checkout)

Thanks in advance!

**Comment by danieljimeneznz** [comment]
Hi @taraghi as part of some work I did for a client at my company I added tagging to releases on a [fork we maintain](https://github.com/elementx-ai/moodle-local-oauth). We are however unable to update to the latest release as [v1.12.0](https://github.com/bshaffer/oauth2-server-php/releases/tag/v1.12.0) of the 3rd party plugin drops support for PHP 7.0 which means that people running Moodle 3.x would be unable to use the plugin (although the changes only seem to change tests and development dependencies which wouldn't effect the plugin - so I might upgrade it at some point :thinking:). I have however tried to rectify an issue that a customer was seeing with the `count` operator via https://github.com/elementx-ai/moodle-local-oauth/pull/23 in an attempt to get this to work with Moodle 4.x.
